Header/Header.js

```diff
@@ -105,6 +113,11 @@ const HeaderBase = kind({
 	},
 
 	computed: {
+		'aria-label': ({title, subtitle}) => {
+			return subtitle ?
```

Comment:
It seems `aria-label` is public API but you replace the prop with this.
Did you intend this?

Reply:
I thought that we would support custom aria-label in another issue. The `aria-label` is not a  public API now. But the behavior has broken as you mentioned.

Anyway, as you mentioned it, I fixed it.